src/derivatives/atlas.md

```diff
@@ -8,45 +8,43 @@ segmentations, and other knowledge annotations such as landmarks defined with
+     and the template is NOT serving as the authoritative definition of a space.
 -    [`cohort-<label>`](../glossary.md#cohort-entities) is REQUIRED to disambiguate derivatives defined with
-     respect to different cohort instances of a single [space (coordinate system)](../appendices/coordinate-systems.md).
+     respect to different cohort instances.
      Please note that [`cohort-<label>`](../glossary.md#cohort-entities) MUST NOT be used if neither
-     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities) are used.
--    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to encode files pertaining
-     or derived from the atlas identified by the entity's label.
+     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities)
+     nor [`atlas-<labl>`](../glossary.md#atlas-entities) are used.
 -    [`seg-<label>`](../glossary.md#segmentation-entities) is REQUIRED when a single atlas has several different
      realizations (for instance, segmentations and parcellations created with different criteria) that
      need disambiguation.
 -    [`scale-<label>`](../glossary.md#scale-entities) is REQUIRED to disambiguate different atlas 'scales',
      when the atlas has more than one 'brain unit' resolutions, typically relating to the area covered
      by regions.
 
-The general filename pattern for subject derivatives with templates and atlases in their provenance
+The general filename pattern for subject-level derivatives with templates and atlases 
 follows the general BIDS-Derivatives pattern:
 
 ```Text
 <pipeline_name>/
     sub-<label>/
         <datatype>/
-            <source_entities>[_space-<space>][_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
+            <source_entities>[_atlas-<label>][_space-<space>][_cohort-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
+            <source_entities>[_tpl-<label>][_space-<space>][_cohort-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
 ```
 
-[`atlas-<label>`](../glossary.md#atlas-entities), [`seg-<label>`](../glossary.md#segmentation-entities),
-and [`scale-<label>`](../glossary.md#scale-entities) are discussed later in section
-[Filenames of derivatives with atlases in their provenance](#filenames-of-derivatives-with-atlases-in-their-provenance).
-
 For derivatives of template- and altas-generating pipelines, which typically aggregate
 several sessions and/or subjects, the derivatives-specific
-[`tpl-<label>` entity](../glossary.md#template-entities) is dual in terms of usage to BIDS raw's
+[`tpl-<label>` entity](../glossary.md#template-entities) and
+[`atlas-<label>` entity](../glossary.md#atlas-entities) are dual in terms of usage to BIDS raw's
```

Comment:
```suggestion
[`tpl-<label>` entity](../glossary.md#template-entities) is dual in terms of usage to BIDS raw's
```
This confronts the soul of my proposal:

sub <-> tpl are the two faces of the same coin, sub for individual, tpl for a group of individuals
session <-> cohort are the two faces of the same coin, subjects have sessions, templates have (not as close analog as the coordinate system, but still analog) cohorts.

atlas is special in that it represents a construct of prior knowledge with reference to a specific authoritative coordinate system.